src/api-v1.js

```diff
@@ -90,10 +91,12 @@ api.declare({
     let result;
 
     do {
+      let startTime = new Date();
```

Comment:
I definitely like this idea.  we've wrapped timed calls in docker-worker since the early influx days to produce timings in this way.